spec/integration/elasticsearch/transform/transform.spec.yml

```
@@ -34,10 +34,11 @@ spec:
           - type: array
             items:
               type: string
+              pattern: "^[^,]+$"
           - type: string
```

Nit. Maybe the pattern below should go here.
```suggestion
          - type: string
            pattern: "^[^,]+$"
```